src/VisualStudio/Core/Impl/CodeModel/InternalElements/AbstractCodeType.cs

```diff
@@ -123,7 +130,7 @@ public EnvDTE.CodeNamespace Namespace
         {
             get
             {
-                var namespaceNode = GetNamespaceOrTypeNode();
```

Comment:
[This](http://index/Microsoft.VisualStudio.LanguageServices.Implementation/R/6965f7e8fbf3f951.html) uses `GetNamespaceOrTypeNode()`